src/components/Modals/UseAsAliasModal/UseAsAliasModal.types.ts

```diff
@@ -0,0 +1,20 @@
+import { Dispatch } from 'redux'
+import { ModalProps } from 'decentraland-dapps/dist/providers/ModalProvider/ModalProvider.types'
+import { ENS } from 'modules/ens/types'
+
+export type Props = ModalProps & {
+  address?: string
+  isLoading: boolean
+  usedAsAlias: ENS[]
```

Comment:
This prop is here to represent the aliases a user has? if so, 
- is it posible for a user to have more than one alias?
- If not, what do you think about renaming this to `alias?: ENS`? if they can, maybe this could be re-named `aliases`? to be more succinct. What do you think?

Reply:
Agree! I understand that the user in the interace can't have more than one avatar, but our code is prepared to have more than one. So, I'll prefer to follow with the Multiple Avatars option (then, multiple aliases) and preselet the [0] of the array as we made before. What do you think? In the mean team I'll re-name to `aliases`